### PR TITLE
WRQ-20844: Fixed React 18.3 warnings in agate

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In React 19, `defaultProps` in function components will be removed. 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Removed `defaultProps` for function components and replaced them with ES6 default parameters.
Passed key attribute directly to JSX without using spread

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
There is an issue with travis and mongoDB, so I appllied a fix on travis.yml

### Links
[//]: # (Related issues, references)
WRQ-20844

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: VirtualList/VirtualList.js [hunk 1]
```diff
@@ -30,8 +30,50 @@ const nop = () => {};
  * @ui
  * @public
  */
-let VirtualList = ({itemSize, role, ...rest}) => {
-	const props = itemSize && itemSize.minSize ?
+// let VirtualList = ({itemSize, role, ...rest}) => {
```
Comment (adrian-cocoara-lgp) [inline VirtualList/VirtualList.js:33]: remove this commented line

File: VirtualList/VirtualList.js [hunk 2]
```diff
@@ -480,7 +504,45 @@ VirtualList.defaultProps = {
  * @ui
  * @public
  */
-let VirtualGridList = ({role, ...rest}) => {
+// let VirtualGridList = ({role, ...rest}) => {
```
Comment (adrian-cocoara-lgp) [inline VirtualList/VirtualList.js:507]: remove commented line

File: samples/sampler/package.json
Comment (juwonjeong) [inline samples/sampler/package.json:1]: Make "@enact/storybook-utils": "^5.1.4",

File: Scroller/Scroller.js [hunk 3]
```diff
@@ -45,8 +45,39 @@ const nop = () => {};
  * @public
  */
 let Scroller = (props) => {
-	// Hooks
+	const {
```
Comment (juwonjeong) [inline Scroller/Scroller.js:48]: Could you please check changes of Scroller and VirtualList in https://github.com/enactjs/sandstone/pull/1613/files?
With the current implementation we cannot load defaultProp values properly in Scroller sampler.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**